### DISCUSSION TITLE
申请退款，实在无法使用

### DISCUSSION THREAD

**Discussion by CaesarH199** [getlantern/lantern-forum-cn#993] [General]
买了两年的专业版，付费过后根本难以使用，电脑版正常，但是手机版只要连接wifi就无法使用，用流量也是时好时坏，凑合着用了一年，反馈了无数次，不但没有改善，还越来越差劲了，是在无法忍受了，把剩余的一年退给我，不再继续使用了

**Comment by CaesarH199** [comment]
![1729825588215](https://github.com/user-attachments/assets/17707b53-c338-4356-8970-04874d7eb5a0)
![Screenshot_2024-10-25-11-06-19-05_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/user-attachments/assets/7731731e-7aa4-419f-83c5-c5c9e4b0427a)
信用卡付款凭证以及蓝灯账号

**Reply by CaesarH199** [reply]
@Mike777Khoo

**Reply by CaesarH199** [reply]
因为节点位置不断跳动，导致我多个账号被封，现在隧道分流功能也出现问题，完全无法使用

**Comment by CaesarH199** [comment]
![Screenshot_2024-10-25-11-35-27-84_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/a3db0e2e-d2c8-44fb-a557-1b05d6e4f306)
![Screenshot_2024-10-25-11-35-48-60_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/4479836e-8624-4895-ac44-028eac8014d9)
![Screenshot_2024-10-25-11-35-56-52_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/9b2fb459-6518-4e53-9a32-f7b1039143b7)
无数次反馈全是机器人回复，早就想退款找不到退款方法

**Reply by CaesarH199** [reply]
没人处理吗

**Comment by MaxMKMax** [comment]
@CaesarH199 您好. 已经帮您提交上去了. 要等待支付部门处理. 不好意思让您久等了. 谢谢!

**Reply by CaesarH199** [reply]
大概需要多久呢？

**Reply by CaesarH199** [reply]
@Mike777Khoo 大概多久会有结果？

**Reply by MaxMKMax** [reply]
@CaesarH199 请您给我们一周. 我这里还等待回复呢. 不好意思啊.

**Reply by CaesarH199** [reply]
@Mike777Khoo 有回复了吗

**Reply by MaxMKMax** [reply]
@CaesarH199 我帮您问一下.

**Reply by CaesarH199** [reply]
@Mike777Khoo 这么久过去了，请问结果如何了

**Reply by CumanMercenary** [reply]
> @Mike777Khoo这么久过去了，请问结果如何了

哥，拿到退款了吗

**Reply by CaesarH199** [reply]
没有

**Comment by greySkiesMarsh** [comment]
Hi @CaesarH199 

I'm sorry for the delay in resolving this issue. 

Unfortunately, we aren't able to approve this refund request. I encourage you to please try the latest Lantern release to see if your connectivity issues have been resolved by the latest fixes deployed by our engineers, but your purchase was made too long ago to be eligible for a refund per the payment processor's policy.

**Reply by CaesarH199** [reply]
说我购买时间太久了无法退款？你们app上根本就没有退款的渠道，刚买两天的时候我就想退款了，那时候根本不知道在哪申请，你们的产品什么样自己没点数吗？每次都是让下载最新版本，你版本再新你就是有问题有个屁用？好好说没用非要投诉闹大才行是吗？

**Reply by CaesarH199** [reply]
![Screenshot_2024-11-29-11-26-39-98_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/810866d4-285e-4b58-8810-5a11be7a50db)
![Screenshot_2024-11-29-11-26-34-24_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/88314cc2-510b-4ac7-9a8d-5931241f5a23)
![Screenshot_2024-11-29-11-26-26-84_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/9a68fcf9-eb87-4247-9f15-3f3e9ad717f6)
![Screenshot_2024-11-29-11-26-21-50_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/76499173-104d-41d4-8085-64f786de43fd)
![Screenshot_2024-11-29-11-26-17-34_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/d42d057c-5581-42ee-942c-694d5756b4e2)
![Uploading Screenshot_2024-11-29-11-26-13-22_e307a3f9df9f380ebaf106e1dc980bb6.jpg…]()

**Reply by CaesarH199** [reply]
每天投诉八百遍，换来的就是机器回复，客服客服没有，付了钱之后就是自助用产品，没办法用不解决，根本没有任何服务，这样的做法和骗子公司有什么两样？

**Reply by CaesarH199** [reply]
@Mike777Khoo @greySkiesMarsh 所以你们最后结果是怎么样呢？
